source/Internal/Framework/Theme/Command/ClearCacheCommand.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OxidEsales\EshopCommunity\Internal\Framework\Theme\Command;
```

Comment:
Maybe move it to a different namespace since it does not only clear the theme-cache files?

Reply:
The container_cache.php is being generated into the tmp directory.

See:
https://github.com/OXID-eSales/oxideshop_ce/blob/master/source/Internal/Transition/Utility/BasicContext.php#L36

Reply:
The command now supports cache folder clearing